karyon3-admin-simple/src/main/java/com/netflix/karyon/admin/HttpServerConfig.java

```diff
@@ -0,0 +1,25 @@
+package com.netflix.karyon.admin;
+
+import com.netflix.archaius.annotations.Configuration;
+import com.netflix.archaius.annotations.DefaultValue;
+
+@Configuration(prefix="karyon.server.admin")
+public interface HttpServerConfig {
+    @DefaultValue("null")
```

Comment:
Does this get converted into a actual null value someplace?  Is it possible to omit a default value?